templates/solutions/infrastructure/_form_infrastructure.html

```diff
@@ -0,0 +1,89 @@
+<div class="p-modal" style="display: none;" id="infrastructure-modal">
```

Comment:
No changes here, the file has just been moved to a different folder. No need to review